lib/jenkins_api_client/node.rb

```diff
@@ -137,7 +137,7 @@ def create_dumb_slave(params)
         @logger.debug "Creating a dumb slave with params: #{params.inspect}"
         default_params = {
           :description => "Automatically created through jenkins_api_client",
-          :executors => 2,
+          :executors => 6,
```

Comment:
Let's not change this default value since it may be breaking anybody relying on this default behavior.